plugins/scaffolder-backend/api-report.md

```diff
@@ -710,4 +710,8 @@ export class TemplateActionRegistry {
 
 // @public (undocumented)
 export type TemplateFilter = (...args: JsonValue[]) => JsonValue | undefined;
+
+// Warnings were encountered during analysis:
+//
+// src/scaffolder/actions/builtin/publish/gitlabMergeRequest.d.ts:16:9 - (tsdoc-missing-deprecation-message) The @deprecated block must include a deprecation message, e.g. describing the recommended alternative
```

Comment:
Probably want to fix this comment too, basically it needs a deprecation reason after the deprecation. :)

```ts
/** @deprecated please remove the use of this parameter as it's no longer required */
```

Reply:
can apply that to all the deprecated ones apart from the output, say that you should use `projectPath` instead?

Reply:
My bad! My first deprecating something 🛩️